package.json

```diff
@@ -35,8 +34,8 @@
     "rebuild": "npm -w @lavamoat/lavapack -w @lavamoat/viz -w @lavamoat/yarn-plugin-allow-scripts run rebuild",
     "lint:staged": "lint-staged",
     "lint:commit": "commitlint",
-    "postinstall": "husky install",
-    "prepublishOnly": "cross-env NODE_ENV=production npm run rebuild && npm run test:prep && npm run test",
+    "postinstall": "husky install; npm run rebuild",
+    "prepublishOnly": "npm run --omit=dev rebuild && npm run test:prep && npm run test",
```

Comment:
> * removes `cross-env`; it is not needed to set `NODE_ENV=production` when using `npm run` since `npm run --production` _sets it for you_

do we not want to add the `--production` flag here?

to preserve setting `NODE_ENV=production`

Reply:
`--production` is deprecated. `--omit=dev` is how it's expressed now. 

(note that this is not a choice I would have made.)